test_programs/compile_success_with_bug/underconstrained_value_detector_5425/Nargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "underconstrained_value_detector_5425"
```

Comment:
I see this `compile_success_with_bug` directory is new. I don't think CI will do anything with it as it's not handled in [build.rs](https://github.com/noir-lang/noir/blob/d86891bea9e01d12988d4ee344883d018be32ea7/tooling/nargo_cli/build.rs#L32-L39). Perhaps `generate_compile_success_no_bug_tests` could be an example of what to do with it?

Reply:
Oh right. Thank you, working on it

Reply:
Done